ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -90,7 +96,12 @@ class SandboxServer(actorSystemName: String, config: => SandboxConfig) extends A
   // TODO: Pass this info in command-line (See issue #2025)
   val participantId: ParticipantId = Ref.LedgerString.assertFromString("sandbox-participant")
 
-  private val authService: AuthService = AuthServiceWildcard
+  // FIXME If this code is still here when reviewing, please ask me to remove it
+  private val authService: AuthService = AuthServiceStatic {
+    case "root" => Claims.wildcard
+    case "admin" => Claims(Seq(ClaimAdmin))
```

Comment:
Add `ClaimPublic`, otherwise you won't be able to use "public" APIs. Same below.

Reply:
Ah, I thought public was equal to non-empty. Shouldn't we make it that way?